api/middleware/authorization.ts

```diff
@@ -1,15 +1,18 @@
-import jwt from 'jsonwebtoken';
-import { Request, Response, NextFunction } from 'express';
+import { Response, NextFunction } from 'express';
+import { UserInfoRequest } from './requests';
 
-module.exports = async (request: Request, response: Response, next: NextFunction) => {
+const jwt = require('jsonwebtoken');

```

Comment:
Dlaczego zamieniłeś import za pomoca "import jwt from ..." na require?

Reply:
Chodziło o to co zwraca funkcja verify po zmianie na require błedy się nie pojawiały co do typów, które zwraca funkcja. Zwraca ona string | object. W tym momencie będę typował payload na any.

Reply:
A próbowałeś może zainstalowac dodatkowo typy do jsonwebtoken? Czy to cos pomogło?
Jeżeli nie to zostawmy require